### PR TITLE
[ED-87] [ED-89] - Stock management base configuration for Dalbe

### PR DESCRIPTION
**Pull request by purug**
image

This same PR also allows to create a delivery plan as it takes configurations from ED-87. 

image

image

image

### CODE REVIEW

File: axelor-stock/src/main/resources/demo/fr/stock_stockConfig.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
 "importId";"name";"code";"company.importId";"customerVirtualStockLocation.importId";"supplierVirtualStockLocation.importId";"inventoryVirtualStockLocation.importId";"customsMassUnit.importId";"receiptDefaultStockLocation.importId";"pickupDefaultStockLocation.importId"
-"1";"Axelor";"AXE";1;2;3;4;306;1;1
+"1";"Axelor";"AXE";1;2;3;4;306;7;8
```
Comment (AntoineXev) [inline axelor-stock/src/main/resources/demo/fr/stock_stockConfig.csv]: Why axelor is still here ?

File: axelor-stock/src/main/resources/demo/fr/stock_freightCarrierMode.csv [hunk 2]
```diff
@@ -1,7 +1,3 @@
 "importId";"name";"carrierPartner.importId";"shipmentMode.importId"
-1;"Colissimo";8002;1
-2;"Chronopost";8004;2
-3;"TNT Express";8001;2
-4;"FedEx Home International";8005;3
-5;"La poste Business international solution";8002;3
-6;"Relais colis";8003;4
+1;"DHL";8001;1
```
Comment (AntoineXev) [inline axelor-stock/src/main/resources/demo/fr/stock_freightCarrierMode.csv]: As that are the carrier partners, we could go to only one carrier with generic name: "Livraison fournisseur"

**Review by AntoineXev — CHANGES_REQUESTED** [review]
Small changes

File: axelor-stock/src/main/resources/demo/fr/stock_stockConfig.csv (hunk 1 above)
Comment (purug) [reply axelor-stock/src/main/resources/demo/fr/stock_stockConfig.csv]: missed it, updated now.

File: axelor-stock/src/main/resources/demo/fr/stock_freightCarrierMode.csv (hunk 2 above)
Comment (purug) [reply axelor-stock/src/main/resources/demo/fr/stock_freightCarrierMode.csv]: okay, updated

**Review by AntoineXev — APPROVED** [review]

**Pull request merged**